### PR TITLE
Fix keycloak configmap

### PR DESCRIPTION
**Pull request by kahootali**
Signed-off-by: Ali Kahoot
username: kahootali
email: kahoot.ali@outlook.com

#### What this PR does / why we need it:
It fixes the keycloak configmap which was failing in the latest version as it added the `standalone.xml` to next line rather than previous line.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ x] Chart Version bumped

### CODE REVIEW

File: stable/keycloak/templates/configmap.yaml [hunk 1]
```diff
@@ -20,8 +20,8 @@ data:
     echo 'Running custom pre-start script...'
 {{ . | indent 4 }}
   {{- end }}
-
-    exec /opt/jboss/tools/docker-entrypoint.sh -b 0.0.0.0 {{ .Values.keycloak.extraArgs }}{{- if $highAvailability }} -c standalone-ha.xml{{ else }} -c standalone.xml{{ end }}
+    # Moving standalone.xml earlier as it get moved to next line when templating this chart
+    exec /opt/jboss/tools/docker-entrypoint.sh -b 0.0.0.0 {{ if $highAvailability }} -c standalone-ha.xml{{ else }} -c standalone.xml{{ end }} {{ .Values.keycloak.extraArgs }}
```
Comment (unguiculus) [inline stable/keycloak/templates/configmap.yaml:24]: I fail to see what the issue is and cannot reproduce it. Don't you just move the extra args to the end? Why would that change anything?

Comment (kahootali) [reply stable/keycloak/templates/configmap.yaml:24]: @unguiculus The issue is if I add multiple extraArgs like in values.yaml if I add 
```
  extraArgs: >
    -Djgroups.bind_addr=global
    -Dkeycloak.migration.strategy=IGNORE_EXISTING    
    -Dkeycloak.migration.action=import
```
and I templatize the keycloak chart, it adds the standalone.xml statement in next line like 
```
1. exec /opt/jboss/tools/docker-entrypoint.sh -b 0.0.0.0 -Djgroups.bind_addr=global -Dkeycloak.migration.strategy=IGNORE_EXISTING     -Dkeycloak.migration.action=import
2. -c standalone.xml
```

Comment (unguiculus) [reply stable/keycloak/templates/configmap.yaml:24]: You need to use this:

```yaml
  extraArgs: >-
    -Djgroups.bind_addr=global
    -Dkeycloak.migration.strategy=IGNORE_EXISTING    
    -Dkeycloak.migration.action=import
```

Note the hyphen after `>`.

**Pull request closed without merging**